src/EnergyPlus/Autosizing/Base.hh

```diff
@@ -185,8 +185,8 @@ struct BaseSizer
     // HeatingWaterDesCoilWaterVolFlowUsedForUASizer, HeaterWaterflowSizing
     int dataWaterLoopNum = 0;
     // CoolingWaterflowSizing, CoolingWaterDesAirInletTempSizer
-    int dataFanIndex = -1;
-    int dataFanEnumType = -1;
+    int dataFanIndex = 0;
+    HVAC::FanType dataFanType = HVAC::FanType::Invalid;
```

Comment:
Invalid is much nicer.  And again, I don't have a concern over the 0 or 1 based.  Either way, the uniformity is wonderful.

Reply:
The uniformity is not just between Fans and HVACFans, it's between Fans and all other AirLoop components. If all other components use 1-based component indices, then using 0-based for fans makes things hairy. We can move to 0-based once we can do it for all AirLoop components.

Reply:
My first cut at this actually used `std::vector` for the `fans` pointer array. It made AirLoop stuff messy, so I junked it and started over with `Array1D`.